pack/tic/lif_encounter.json

```diff
@@ -0,0 +1,578 @@
+        "clues": 1,
+        "code": "07250",
+        "double_sided": true,
+        "encounter_code": "a_light_in_the_fog",
+        "encounter_position": 20,
+        "faction_code": "mythos",
+        "illustrator": "Dual Brush Studios",
+        "name": "Sunken Archives",
+        "pack_code": "lif",
+        "position": 250,
+        "quantity": 1,
+        "shroud": 2,
+        "text": "Clues cannot be discovered at Sunken Archives while it is flooded.\n<b>Forced</b> - After Sunken Archives is revealed: It becomes fully flooded. Place the set-aside black key on it.",
+        "traits": "Cave.",
+        "type_code": "location",
+        "victory": 1
+    },
+    {
+        "back_flavor": "The air is frigid here, and there is no sound but the faint plinking of water dripping onto the rough stone ground. You shudder and proceed onward...",
+        "back_illustrator": "Patrick McEvoy",
+        "back_name": "Tidal Tunnel",
+        "clues": 0,
+        "code": "07251",
+        "double_sided": true,
+        "encounter_code": "a_light_in_the_fog",
+        "encounter_position": 21,
+        "faction_code": "mythos",
+        "illustrator": "Álvaro Calvo Escudero",
+        "name": "Pump Room",
+        "pack_code": "lif",
+        "position": 251,
+        "quantity": 1,
+        "shroud": 3,
+        "text": "[action]: Test [intellect] (2) or [agility] (2). If you succeed, choose two locations. Decrease the flood level of one location and increase the flood level of the other (if you succeed by 2 or more, you may instead choose just one location and decrease its flood level).",
+        "traits": "Cave.",
+        "type_code": "location"
+    },
+    {
+        "back_link": "07252b",
+        "code": "07252",
+        "encounter_code": "a_light_in_the_fog",
+        "encounter_position": 22,
+        "faction_code": "mythos",
+        "name": "Captured!",
+        "pack_code": "lif",
+        "position": 252,
+        "quantity": 1,
+        "text": "<b>The first time an investigator is captured, read the following:</b>\n<i>You are bound and dragged deep underground, then thrown into a sealed prison cell, surrounded on all sides by dripping stone walls. Your cries fall on deaf ears as your captor's footsteps fade into the distance.</i>\nRemove the captured investigator from their location and return each asset in that investigator's hand slots to their hand.\nTurn that investigator's mini-card sideways to indicate that they have been captured. Captured investigators cannot move, fight, or be engaged.\nFlip this story card over to its \"Holding Cells\" side and put it into play. It is not on any existing row or column, and is only connected to Sunken Grotto locations through its connection icons. Place the captured investigator in Holding Cells.\nFrom now on, whenever an investigator is captured, resolve this same effect (return each asset in their hand slots to their hand, turn their mini-card sideways, and place them in Holding Cells).",
```

Comment:
I believe the flavor here should have the `<blockquote>` treatment around the `<i>` tag?